src/BasicStepperDriver.cpp

```diff
@@ -102,3 +108,18 @@ int BasicStepperDriver::rotate(double deg){
     int steps = deg * motor_steps * microsteps / 360;
     return move(steps);
 }
+
+/*
+ * Enable/Disable the motor by setting a digital flag
+ */
+void BasicStepperDriver::enable(void){
+    pinMode(enable_pin, OUTPUT);
```

Comment:
Setting up pin mode should be done in init, so it's not floating. The only question is should it be enabled or disabled at start.